example-todomvc/build.sbt

```diff
@@ -0,0 +1,15 @@
+enablePlugins(ScalaJSPlugin, ScalaJSBundlerPlugin)
```

Comment:
fix incoming, but as the task is not the same, I'm not sure if/how travis.yml needs to be modified (haven't used travis before); here's how to build the project now: `"; project example-todomvc; fastOptJS::webpack"`.